openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -72,15 +73,18 @@ const Carousel = {
             slidesToScroll: a,
             responsive: responsive_settings
         });
-
+        //This loads in i18n strings from a hidden input element, generated in the books/custom_carousel.html template.
+        const i18n_values = JSON.parse($('input[name=\'carousel-i18n-strings\']').attr('value'))
```

Comment:
```suggestion
        const i18nValues = JSON.parse($('input[name="carousel-i18n-strings"]').attr('value'))
```
Use camel case for JS variable names.  Replace escaped single quotes with double quotes.